src/materialized/tests/sql.rs

```diff
@@ -503,25 +503,40 @@ fn test_tail_shutdown() -> Result<(), Box<dyn Error>> {
     ore::test::init_logging();
 
     let temp_dir = tempfile::tempdir()?;
-    let (_server, mut client) = util::start_server(util::Config::default())?;
-
-    // Create a tailing file source that never produces any data. This is the
-    // simplest way to cause a TAIL to never terminate.
-    let path = Path::join(temp_dir.path(), "file");
-    fs::write(&path, "")?;
-    client.batch_execute(&*format!(
-        "CREATE MATERIALIZED SOURCE s FROM FILE '{}' WITH (tail = true) FORMAT BYTES",
-        path.display()
-    ))?;
-
-    // Launch the ill-fated tail.
-    client.copy_out("COPY (TAIL s) TO STDOUT")?;
+    let (server, _) = util::start_server(util::Config::default())?;
+
+    // We have to use the async PostgreSQL client so that we get ungracefully
```

Comment:
I'm not sure I understand this comment. I think you want to s/get/can but maybe not?

Reply:
Thanks, fixed!